Assets/Scripts/PickupStaminaBoost.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PickupStaminaBoost : MonoBehaviour {
```

Comment:
Better design would be to create a `PickUpBoost` base class which implements `Pickup` and `ApplyEffect` behavior and then make `PickupStaminaBoost` inherit that